### ISSUE TITLE
[Bug] Bambu Lab A1 offline

### ISSUE THREAD

**Issue by qbahc** [greghesp/ha-bambulab#693]
### Describe the bug

After update Ha-bambulab A1 gone offline permanently (in HA, Bambu Handy works ok, A1 online).
Before update was ok.
When I try force data regresu i got error:

![Screenshot_2024-11-06-18-29-19-703_io homeassistant companion android-edit](https://github.com/user-attachments/assets/f9873fd6-f150-4c58-96e8-a84988fb66ca)

### To Reproduce

Just all HA-bambulab Integration entities are down

### Expected Behaviour

A1 online, ha bambulab entities on-line etc.

### What device are you using?

A1

### Diagnostic Output

```shell
{
  "home_assistant": {
    "installation_type": "Home Assistant OS",
    "version": "2024.10.4",
    "dev": false,
    "hassio": true,
    "virtualenv": false,
    "python_version": "3.12.4",
    "docker": true,
    "arch": "aarch64",
    "timezone": "Europe/Warsaw",
    "os_name": "Linux",
    "os_version": "6.6.31-haos-raspi",
    "supervisor": "2024.11.2",
    "host_os": "Home Assistant OS 13.2",
    "docker_version": "27.2.0",
    "chassis": "embedded",
    "run_as_root": true
  },
  "custom_components": {
    "tuya_local": {
      "documentation": "https://github.com/make-all/tuya-local",
      "version": "2024.11.0",
      "requirements": [
        "tinytuya==1.15.1",
        "tuya-device-sharing-sdk>=0.1.9,<=0.2.99"
      ]
    },
    "dreame_vacuum": {
      "documentation": "https://github.com/Tasshack/dreame-vacuum",
      "version": "v2.0.0b16",
      "requirements": [
        "pillow",
        "numpy",
        "pybase64",
        "requests",
        "pycryptodome",
        "python-miio",
        "py-mini-racer",
        "paho-mqtt"
      ]
    },
    "smartlife": {
      "documentation": "https://www.home-assistant.io/integrations/smartlife",
      "version": "0.1.0",
      "requirements": [
        "tuya-device-sharing-sdk==0.2.0"
      ]
    },
    "xiaomi_cloud_map_extractor": {
      "documentation": "https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor",
      "version": "v2.2.0",
      "requirements": [
        "pillow",
        "pybase64",
        "python-miio",
        "requests",
        "pycryptodome"
      ]
    },
    "dwains_dashboard": {
      "documentation": "https://dwainscheeren.github.io/dwains-lovelace-dashboard/",
      "version": "3.7.1",
      "requirements": []
    },
    "tuya_v2": {
      "documentation": "https://github.com/tuya/tuya-home-assistant",
      "version": "1.5.0",
      "requirements": [
        "tuya-iot-py-sdk==0.4.1"
      ]
```

**Comment by kanflo** [comment]
This is the same issue as #689 and #692.

**Comment by AdrianGarside** [comment]
Should be fixed with #698
